src/styles.scss

```diff
@@ -1,6 +1,30 @@
-/* You can add global styles to this file, and also import other style files */
 @import "~bootstrap/dist/css/bootstrap.css";
 @import "./scss/mixins";
 @import "./scss/variables";
 @import "./scss/fonts";
 @import "./scss/reset";
+
+.spinner {
```

Comment:
@dnistrova I meant a bit vice-versa. Move all styles related to the spinner to loader.component.scss and delete here in src/styles.scss